File: combine-handler/src/main/java/cc/vileda/kalfor/handler/KalforRequest.java
package cc.vileda.kalfor.handler;

import java.util.List;


public class KalforRequest
{
	String proxyBaseUrl;
	List<KalforProxyRequest> proxyRequests;

	@SuppressWarnings("unused")
	public KalforRequest()
	{
	}

	public KalforRequest(final String proxyBaseUrl, final List<KalforProxyRequest> proxyRequests)
	{
		this.proxyBaseUrl = proxyBaseUrl;
		this.proxyRequests = proxyRequests;
	}
}

File: library/src/main/kotlin/cc/vileda/kalfor/handler/KalforRequest.kt
package cc.vileda.kalfor.handler

import io.vertx.core.http.HttpMethod


data class KalforRequest(
        var proxyBaseUrl: String = "",
        var headers: List<KalforProxyHeader> = mutableListOf(),
        var proxyRequests: List<KalforProxyRequest> = mutableListOf(),
        val type: HttpMethod = HttpMethod.POST
)
